getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -47,6 +47,21 @@ you can download just the
 instead.
 Make sure you at least have the .NET Framework 4.5 targeting pack installed, you can get it using any of the installers mentioned above inside the "Individual components" tab.
 
+Windows (JetBrains Rider)
+~~~~~~~
```

Comment:
Number of tildes should match the heading. Also, add a blank line after the heading:

```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~

```